css/types/image.json

```diff
@@ -1768,14 +1768,10 @@
               "edge": "mirror",
               "firefox": {
                 "version_added": false,
-                "notes": [
-                  "See <a href='https://bugzil.la/703217'>bug 703217</a>.",
-                  "The <a href='https://developer.mozilla.org/docs/Web/CSS/-moz-image-rect'><code>-moz-image-rect()</code></a> function supports fragments as of Firefox 4."
```

Comment:
Is the removal of these notes a manual change?

Reply:
Yes